x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_definitions/entity_descriptions/host.ts

```diff
@@ -46,4 +46,12 @@ export const hostEntityEngineDescription: EntityDescription = {
     collect({ source: 'host.architecture' }),
     ...getCommonFieldDescriptions('host'),
   ],
+  customIngestProcessors: [
```

Comment:
This comment goes for user and host.

Right now, there is no way to express user type and host type. To simplify at this moment, I agree with hardcoding Host and Identity always.

But the near future once we have the nested `entity` type under `host.*`  and `user*` I would have this processor with `override: false` so we don't override any manual type used.

So this is mostly a reminder for us @uri-weisman , this will need to be updated.